src/Extractor/ConfluenceExtractor.php

```diff
@@ -30,6 +35,10 @@ protected function doExtract( SplFileInfo $file ): bool {
 		$this->dom->load( $file->getPathname() );
 		$this->helper = new XMLHelper( $this->dom );
 
+		if ( isset( $this->config['config']['categories'] ) ) {
+			$this->categories = $this->config['config']['categories'];
+		}
```

Comment:
Would it make sense to allow different categories for different spaces?

Not necessary though...

Reply:
No, not now